Topic4/src/interfaces/User.java

```diff
@@ -0,0 +1,24 @@
+package interfaces;
+
+import java.awt.Image;
+
+public interface User {

```

Comment:
Which was the intention to create this interface? Wouldn't be better to have a class "User" with all the attributes and methods and then be used by WebUser instead of implement it?